src/lib/overloads.jl

```diff
@@ -240,7 +240,7 @@ the use of `copy`, manual update, and `convert`.
 function Base.similar(layer::T) where {T <: SimpleSDMLayer}
    emptygrid = similar(layer.grid)
    for i in eachindex(emptygrid)
-      emptygrid[i] = isnothing(layer.grid[i]) ? nothing : zero(eltype(layer.grid))
+      emptygrid[i] = isnothing(layer.grid[i]) ? nothing : zero(eltype(layer.grid[i]))
```

Comment:
`similar()` didn't work without this for climate layers & all. `eltype(temperature.grid)` is always `Union{Nothing, Float32}`, which has no method for `zero()`